source/Calamari/Deployment/PackageRetention/CommandJournalDecorator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Calamari.Commands.Support;
+using Calamari.Common.Plumbing.Deployment.PackageRetention;
+using Calamari.Common.Plumbing.Extensions;
+using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Variables;
+
+namespace Calamari.Deployment.PackageRetention
+{
+    public class CommandJournalDecorator : ICommandWithArgs
```

Comment:
type of decorator - type its decorating - decorator
`PackageJournalCommandDecorator`